src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash/debounce';
 
-export default function debounce(wait = 500, options?: DebounceSettings) {
+export interface DebounceSettings {
```

Comment:
[ts] Module '"/Users/utkarshsaxena/src/github.com/Shopify/javascript-utilities/node_modules/@types/lodash/debounce/index"' resolves to a non-module entity and cannot be imported using this construct.